packages/terra-clinical-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Consumer } from 'xfc';
+import styles from './EmbeddedContentConsumer.scss';
+
+const propTypes = {
+  /**
+   * The source URL of the content to load.
+   */
+  src: PropTypes.string.isRequired,
+  /**
+  * A set of event handlers keyed by the event name.
+  */
+  eventHandlers: PropTypes.arrayOf(PropTypes.shape({
+    key: PropTypes.string,
+    handler: PropTypes.func,
+  })),
+};
+
+class EmbeddedContentConsumer extends React.Component {
+
+  componentDidMount() {
+    Consumer.init();
```

Comment:
Consumer.init() should be called only once in your application, so you might want to put it in a higher level component. You probably don't want to call it for every embedded content.